docs/src/pages/components/breadcrumbs/ActiveLastBreadcrumb.js

```diff
@@ -11,18 +11,23 @@ export default function ActiveLastBreadcrumb() {
   return (
     <div role="presentation" onClick={handleClick}>
       <Breadcrumbs aria-label="breadcrumb">
-        <Link color="inherit" href="/">
+        <Link underline="hover" color="inherit" href="/">
           Material-UI
         </Link>
-        <Link color="inherit" href="/getting-started/installation/">
+        <Link
+          underline="hover"
+          color="inherit"
+          href="/getting-started/installation/"
+        >
           Core
         </Link>
         <Link
+          underline="hover"
```

Comment:
If we've identified that `"always"` is the correct choice, why change it here?

Reply:
`always` is meant to allow links to pops from paragraphs (so you don't have to hover every single word to figure out if it's a link or not. The breadcrumbs pattern is easily identifiable and a lot shorter, it doesn't need the same affordance.

Reply:
That makes a lot of sense. Really appreciate the explainer :+1: